cms/envs/common.py

```diff
@@ -639,6 +639,11 @@
 OPTIMIZELY_PROJECT_ID = None
 OPTIMIZELY_FULLSTACK_SDK_KEY = None
 
+######################## ALGOLIA SEARCH ###########################
+ALGOLIA_APP_ID = None
+ALGOLIA_SEARCH_API_KEY = None
+ALGOLIA_COURSES_RECOMMENDATION_INDEX_NAME = ''
+
```

Comment:
Recommendations in cms don't make sense. I think we can remove these settings from here.